app/models/generic_event/move_redirect.rb

```diff
@@ -30,5 +30,13 @@ def for_feed
         common_feed_attributes['details']['move_type'] = move_type if move_type.present?
       end
     end
+
+    def self.from_event(event)
+      new(event.generic_event_attributes.merge(
+            details: {
+              to_location_id: event.event_params&.dig(:relationships, :to_location, :data, :id),
```

Comment:
the Feed Json in the Google docs called "Events",
states that details should be:
```"details": {
       "to_location_type": "prison",
       "to_location": "HMPEP",
       “move_type”: “court_appearance”
  }
```
Chatting with you @willfish , you seemed in favour of using the `to_location_id`, as it appears in this line/

Reply:
Slight confusion, here. The json you're referencing in your comment is the denormalized feed for the hub. The model itself always has the actual uuid/reference to the corresponding location. And then when we serve the model up for the hub we use the following method to denormalize the location:

https://github.com/ministryofjustice/hmpps-book-secure-move-api/blob/d1fcdf4674696e65a35135e55828a5176d170df3/app/models/location.rb#L70-L78